.gitattributes

```diff
@@ -1,9 +1,3 @@
-.docker              export-ignore
-.github              export-ignore
-.editorconfig        export-ignore
-.gitattributes       export-ignore
-.gitignore           export-ignore
-.php-cs-fixer.php    export-ignore
-.phpstan.neon        export-ignore
-.scrutinizer.yml     export-ignore
-.docker-compose.yml  export-ignore
+.* export-ignore
```

Comment:
```suggestion
.* export-ignore
docker-compose.yml export-ignore
```

I think we should ignore the `docker-compose.yml` as well. What do you think?

Reply:
Indeed.
Thank you.